### PR TITLE
Fixed UI tests in test_subscription.py

### PR DESCRIPTION
**Pull request by ColeHiggins2**
A few tests in ui/test_subscription.py were failing due to:
1. New error message( No longer just a 404 pop-up)
2. non-admin users require more permissions to interact with the manifest modal

### CODE REVIEW

**Review by mshriver — APPROVED** [review]

**Comment by mshriver** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_subscription.py

File: tests/foreman/ui/test_subscription.py [hunk 1]
```diff
@@ -187,7 +201,11 @@ def test_positive_access_with_non_admin_user_with_manifest(test_name):
     org = entities.Organization().create()
     manifests.upload_manifest_locked(org.id)
     role = entities.Role(organization=[org]).create()
-    create_role_permissions(role, {'Katello::Subscription': ['view_subscriptions']})
+    # create_role_permissions(role, {'Katello::Subscription': ['view_subscriptions']})
```
Comment (vsedmik) [inline tests/foreman/ui/test_subscription.py:204]: Is this forgotten or do you really want to keep the comment in?

**Review by vsedmik — COMMENTED** [review]
`test_positive_end_to_end` is still failing in PRT, but it rather seems it needs fix in Airgun.

**Review by latran — APPROVED** [review]
ACK pending one comment.

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**